### PR TITLE
Some changes

### PR DESCRIPTION
**Pull request by kishnel**

### CODE REVIEW

File: DailyGoals/DailyGoals/AddMemberView.swift [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+struct AddMemberView: View {
+
+	@Binding var shouldShow: Bool
+	let onSave: (_ name: String, _ handle: String) -> Void
+
+	@State private var name = ""
+	@State private var handle = ""
+
+	var body: some View {
```
Comment (alissens) [inline DailyGoals/DailyGoals/AddMemberView.swift:18]: spacing needs fixing here

**Review by alissens — CHANGES_REQUESTED** [review]
Why does it feel like the spacing is just off in all the changes?

File: DailyGoals/DailyGoals/Extensions/Array+NSItemProvider.swift [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+extension Array where Element == NSItemProvider {
+
+	func loadObjects(completion: @escaping (Result<URL, Error>) -> Void) -> Bool {
+
+		guard let provider = first else { return false }
+
+		if provider.canLoadObject(ofClass: URL.self) {
+			_ = provider.loadObject(ofClass: URL.self) { object, error in
```
Comment (alissens) [inline DailyGoals/DailyGoals/Extensions/Array+NSItemProvider.swift:21]: let's make discardable?

**Comment by alissens** [comment]
Could you also provide descriptions on your PRs? I can't follow all the changes. 

For example, why icon if no members are found? Why have unknown goal? Our goals are never unknown. 

Things like that.

File: DailyGoals/DailyGoals/AddMemberView.swift (hunk 1 above)
Comment (kishnel) [reply DailyGoals/DailyGoals/AddMemberView.swift:18]: You mean the indentation?

File: DailyGoals/DailyGoals/Extensions/Array+NSItemProvider.swift (hunk 2 above)
Comment (kishnel) [reply DailyGoals/DailyGoals/Extensions/Array+NSItemProvider.swift:21]: loadObject is owned by Apple unfortunately, not our stuff

File: DailyGoals/DailyGoals/AddMemberView.swift (hunk 1 above)
Comment (alissens) [reply DailyGoals/DailyGoals/AddMemberView.swift:18]: yeah

**Review by alissens — APPROVED** [review]

**Pull request merged**